src/libs/actions/Policy.ts

```diff
@@ -2539,6 +2553,99 @@ function setWorkspaceRequiresCategory(policyID: string, requiresCategory: boolea
     API.write('SetWorkspaceRequiresCategory', parameters, onyxData);
 }
 
+function setPolicyRequiresTag(policyID: string, requiresTag: boolean) {
+    const onyxData: OnyxData = {
+        optimisticData: [
+            {
+                onyxMethod: Onyx.METHOD.MERGE,
+                key: `${ONYXKEYS.COLLECTION.POLICY}${policyID}`,
+                value: {
+                    requiresTag,
+                    errors: {
```

Comment:
I think this should be `errorFields`. `errors` is just an object keyed by timestamp, so this wouldn't work as we have here. Same for the others below.

Reply:
errors are working, it shows an error, we have the same in categories
Maybe Im missing something

Reply:
I'm retesting this now

Reply:
No, this is still happening. I think we can continue with this and I'll address it in a fast following PR.